views/index.pug

```diff
@@ -0,0 +1,7 @@
+doctype html
+html(lang="en")
+  head
+    title Form #{method}
+  body
+    p Form with method set to #{method}
+    block content
```

Comment:
Yeah, in retrospect, I could just combine this into the other file, this was made thinking that I would have two different pages